package.json

```diff
@@ -1,28 +1,39 @@
 {
   "name": "universal-search-content",
-  "version": "1.0.0",
-  "description": "the iframe that we put inside the autocomplete dropdown",
+  "version": "0.0.1",
+  "description": "",
   "main": "index.js",
   "scripts": {
+    "build": "./node_modules/.bin/webpack --optimize-minimize --optimize-dedupe",
+    "start": "./node_modules/.bin/webpack-dev-server --content-base dist/",
```

Comment:
I don't think it does, or at least it doesn't work reliably. I had to add `./node_modules/bin` to my `$PATH` in my `.zshrc`